developer/src/common/web/test-helpers/index.ts

```diff
@@ -1,7 +1,10 @@
 import * as fs from 'fs';
-import * as path from 'path';
 import { CompilerEvent, CompilerCallbacks } from '@keymanapp/common-types';
 
+// TODO: schemas are only used by kmc-keyboard for now, so this works at this
+// time, but it's a little fragile if we need them elsewhere in the future
+const SCHEMA_BASE = '../../../../kmc-keyboard/build/src/';
```

Comment:
That's a curious path to have `/build/src/`

Reply:
It relates to how tsc builds -- the build/ folder is the output base folder, then the full source tree is replicated under it (in this case, src/, test/, ...)